frontend/utilities/osquery_tables.ts

```diff
@@ -15,3 +16,33 @@ export const osqueryTables = queryTable.sort((a, b) => {
 export const osqueryTableNames = flatMap(osqueryTables, (table) => {
   return table.hidden ? [] : table.name;
 });
+
+// Note: Hiding columns where table key hidden is set to true
+export const osqueryTableColumnNames = flatMap(osqueryTables, (table) => {
+  const tableColumnNames = flatMap(table.columns, (column) => column.name);
+  return table.hidden ? [] : tableColumnNames;
+});
+
+// Note: Hiding columns where table key hidden is set to true
+export const osqueryTableColumns = flatMap(osqueryTables, (table) => {
+  return table.hidden ? [] : table.columns;
+});
+
+// Note: Hiding columns where table key hidden is set to true or if tables are defined but it doesn't include that table
+export const selectedTableColumns = (selectedTables: string[]) => {
```

Comment:
You can probably also use `filter` here

Reply:
Going to iterate if I find time! Thank you!